.gitattributes

```diff
@@ -1,5 +1,5 @@
 /.github/ export-ignore
-/Build/ export-ignore
+/build/ export-ignore
```

Comment:
What's the intention of the renaming?

Reply:
I tried typo3/tailor yesterday and found that the extension excludes the build folder by default, but not with a capital first letter.
I would like to follow this guideline here. Sooner or later I will also put my own configuration file into the repo to avoid unnecessary files ending up in the TER.

Reply:
Good to know. Thanks for the background information!
I am only wondering now whether it is okay to exclude the source JS Files? Maybe someone wants to include them in an own build process? But that might be a seldom edge case. Someone who wants to do such advanced stuff might not use legacy installation anyway...

Reply:
You are right. I‘m not a developer often build frontend javascript. Perhaps we should move the TypeScript to another folder. What‘s about _Resources/Private/JavaScript_ ?

Reply:
I think that would be okay. That's how https://github.com/jweiland-net/maps2/tree/main/Resources/Private/Build/JavaScript does it.